.ci/jenkins/Jenkinsfile.post-release

```diff
@@ -10,7 +10,7 @@ String stableBranchName = 'stable'
 
 pipeline {
     agent {
-        label 'kie-rhel7'
+        label 'kie-rhel7 && |master'
```

Comment:
A typo, I suppose?
```suggestion
        label 'kie-rhel7 && !master'
```

Reply:
yes - a typo - sorry - will fix this asap